src/ctia/init.clj

```diff
@@ -37,41 +38,50 @@
               (do (newline)
                   (utils/safe-pprint config)))))
 
-(defn default-services
-  "Returns the default collection of CTIA services based on provided properties."
+(s/defn default-services-map
+  :- {(s/constrained s/Keyword simple-keyword?)
+      ;; could be (s/protocol puppetlabs.trapperkeeper.services/ServiceDefinition),
+      ;; but trapperkeeper claims the protocol is internal only
+      (s/pred some?)}
+  "Returns the default map of CTIA services based on provided properties.
+  Keys are the unqualified keyword names of each service protocol, values are
+  the defservice instances."
   [config]
   (let [{auth-service-type :type} (get-in config [:ctia :auth])
         auth-svc
         (case auth-service-type
-          :allow-all allow-all/allow-all-auth-service
-          :threatgrid threatgrid/threatgrid-auth-service
-          :static static-auth/static-auth-service
+          :allow-all {:IAuth allow-all/allow-all-auth-service}
+          :threatgrid {:IAuth threatgrid/threatgrid-auth-service
+                       :ThreatgridAuthWhoAmIURLService threatgrid/threatgrid-auth-whoami-url-service}
```

Comment:
Returning a map here makes it really easy to stub out services. See comment in test helpers for an example.